libs/initScripts/mainnet.js

```diff
@@ -32,10 +32,11 @@ const run = async () => {
       case 'setversion':
         const serviceType = args[4]
         const versionStr = args[5]
+        const dryRun = args[6] || true
```

Comment:
do you need to coerce this to a Boolean in the case of user-provided input 'false' as a String?